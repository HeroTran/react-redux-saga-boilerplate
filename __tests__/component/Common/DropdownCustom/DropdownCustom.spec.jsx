/*
import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import DropdownCustom from '../../../../app/renderer/components/Common/DropdownCustom/DropdownCustom';

let wrapper;
let handleOnChange;
beforeEach(() => {
  handleOnChange = jest.fn();
  const data = [
   {code: "UY", name: "Uruguay"},
   {code: "UZ", name: "Uzbekistan"},
   {code: "VU", name: "Vanuatu"},
   {code: "VA", name: "Vatican City State"},
   {code: "VE", name: "Venezuela"},
   {code: "VN", name: "Vietnam"},
   {code: "VG", name: "Virgin Islands (British)"}
  ];
  const inputValue = "Vietnam";
  const filedName = "countries";
  wrapper = (
    <MemoryRouter>
      <DropdownCustom
        resultData={data}
        inputValue={inputValue}
        filedName={filedName}
        handleOnChange={handleOnChange}
      />
    </MemoryRouter>
  );
});
describe('should be rendered without error', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(wrapper).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
*/