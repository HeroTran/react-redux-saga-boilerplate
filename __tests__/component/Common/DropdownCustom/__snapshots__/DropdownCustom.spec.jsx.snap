// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be rendered without error matches snapshot 1`] = `
<div
  className="dropdown-custom"
>
  <div
    className="widget-input"
  >
    <input
      className="input-text"
      onChange={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      type="text"
      value="Vietnam"
    />
    <i
      className="arrow-down"
      onClick={[Function]}
    />
  </div>
</div>
`;
